NotadogApi/StartupExtentions.cs

```diff
@@ -51,10 +51,17 @@ public static void AddJwt(this IServiceCollection services)
                     {
                         OnMessageReceived = (context) =>
                         {
```

Comment:
This code block becomes dirty. It has repeats and 4 times string literal "Bearer ". Try to use variable or constant for literal and rewrite block using **FirstOrDefault**, [?](https://docs.microsoft.com/ru-ru/dotnet/csharp/language-reference/operators/member-access-operators#null-conditional-operators--and-) and (**Concat** or [??](https://docs.microsoft.com/ru-ru/dotnet/csharp/language-reference/operators/null-coalescing-operator)).